submissions/Vlada-khom/tiny-js-world/index.js

```diff
@@ -0,0 +1,70 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+let man = {
+  entity: "person",
+  name: "John",
+  sex: "male",
+  hands: 2,
+  legs: 2,
+  say: "Hello world!",
+  friends: "Jenny, Rex",
+};
+
+let woman = {
+  entity: "person",
+  name: "Jenny",
+  sex: "female",
+  hands: 2,
+  legs: 2,
+  say: "Hello world! I'm Jenny.",
+  friends: "Tom",
+};
+
+let dog = {
+  entity: "dog",
+  name: "Rex",
+  sex: "male",
+  hands: 0,
+  legs: 4,
+  say: "Gav!",
+  friends: "John, Jenny",
+};
+
+let cat = {
+  entity: "cat",
+  name: "Tom",
+  sex: "male",
+  hands: 0,
+  legs: 4,
+  say: "Meyow!",
+  friends: "Jenny",
+};
+
+// ======== OUTPUT ========
+/* Use print(message) for output.
+   Default tag for message is <pre>. Use print(message,'div') to change containing element tag.
+
+   Message can contain HTML markup. You may also tweak index.html and/or styles.css.
+   However, please, REFRAIN from improving visuals at least until your code is reviewed
+   so code reviewers might focus on a single file that is index.js.
+   */
+
+const arrObjects = [man, woman, cat, dog];
+let list = arrObjects.map((el)=> [el.entity, el.name, el.sex, el.hands, el.legs, el.say, el.friends].join('; '));
+list.forEach(el=> print(el));
```

Comment:
1. Code is written for other people. The closer the code to English the better. Give variables and functions meaningful names. Avoid generic names like `item`, `element`, `key`, `object`, `array` or their variations. Exception: helper functions that are specifically and intentionally designed to be multipurpose (not the case here).
2. `[el.entity, el.name, el.sex, el.hands, el.legs, el.say, el.friends]` is not DRY, contains repetition. As a peer developer I am tasked to add 20 more properties. At inhabitants presentation part of code I want to just list new properties without much repetition.